Week1/homework/app.js

```diff
@@ -1,11 +1,122 @@
 'use strict';
 
-{
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
-  ];
-
-  // Replace with your own code
-  console.log(bookTitles);
+const bookTitles = [
+  'george_orwell_1984',
+  'alexandre_dumas_the_count_of_monte_cristo',
+  'fyodor_dostoyevsky_the_brothers_karamazov',
+  'leo_tolstoy_war_and_peace',
+  'charles_dickens_a_tale_of_two_cities',
+  'john_steinback_of_mice_and_men',
+  'victor_hugo_the_miserables',
+  'balzac_the_lily_of_the_valley',
+  'turgenev_fathers_and_sons',
+  'leo_tolstoy_resurrection',
+];
+
+function generateList() {
+  const createUl = document.createElement('ul');
+  document.body.appendChild(createUl);
+  for (let i = 0; i < bookTitles.length; i++) {
+    const createLi = document.createElement('li');
+    createUl.appendChild(createLi);
+    createLi.setAttribute('id', `${bookTitles[i]}`);
```

Comment:
The setAttribute accepts a string, `bookTitles[i]` already returns a string from the array so there is no need the put it inside a string. 

`createLi.setAttribute('id', bookTitles[i]);`